src/object/cli_obj.c

```diff
@@ -941,6 +943,20 @@ obj_shards_2_fwtgts(struct dc_object *obj, uint32_t map_ver, uint8_t *bit_map,
 						  tgt++, obj_auxi);
 			if (rc != 0)
 				return rc;
+
+			if (update && req_tgts->ort_srv_disp) {
```

Comment:
I think this is for all modify operation? both update/punch?  since it is already check ort_srv_disp, so you do not need update here? 
Hmm, maybe punch does not go through this patch, but if @update is not needed, please remove it?

Reply:
I will update the patch to include punch case. As for the check "ort_srv_disp", we need it, because former check "ort_srv_disp" is out of the scope of current "for" block.